src/components/IOUBadge.js

```diff
@@ -24,8 +24,8 @@ const propTypes = {
 
 const defaultProps = {
     iouReport: {
-        reportID: 0,
-        chatReportID: 0,
+        reportID: '',
+        chatReportID: '',
```

Comment:
@Gonals I know I commented about this before, but what is driving this change of the ids to strings? I am using you updates to the types and I am seeing the type warning in different places. I know there is the project to make reportIDs using a different base but that is not implemented yet. Why should these not be numbers?

cc @marcaaron as I feel like you might have opinion here

Reply:
The main reason is that we are already halfway there. Web is already returning the reportIDs as Strings, so I'm trying to be consistent with that, since we'll eventually have to be.

I'm not seeing warnings anymore, but That's for `fresh` accounts. In accounts where I created IOUs before I added this changes, I get some warnings causes by the old IOUs

Reply:
I see thank you very much for the context!

>  In accounts where I created IOUs before I added this changes, I get some warnings caused by the old IOUs.

Thats probably what I was encountering too